### PR TITLE
Support configuring app2app in portal

### PR DESCRIPTION
**Pull request by tung2744**
ref #3231

Screenshot 2023-08-14 at 2 56 46 PM
Screenshot 2023-08-14 at 2 56 37 PM

@fungc-io Please see if the ui looks good. I cannot find a suitable color in the current figma for some colors such as the status green and grey colors, so I just randomly pick one.

### CODE REVIEW

**Comment by fungc-io** [comment]
@tung2744 please use the same green and blue from the custom domain page
image

otherwise lgtm

File: pkg/lib/config/feature_oauth.go [hunk 1]
```diff
@@ -20,14 +20,16 @@ var _ = FeatureConfigSchema.Add("OAuthClientFeatureConfig", `
 	"additionalProperties": false,
 	"properties": {
 		"maximum": { "type": "integer" },
-		"custom_ui_enabled": { "type": "boolean" }
+		"custom_ui_enabled": { "type": "boolean" },
+		"app2app_enabled": { "type": "boolean" }
```
Comment (louischan-oursky) [inline pkg/lib/config/feature_oauth.go:24]: The server should also respect this config. If app2app_enabled is false, then app2app request should fail with App2AppDisabled error.

**Comment by louischan-oursky** [comment]
The setup link in app2app screen should link to the app2app section directly. It is a bit confusing that it just directs me to the edit client screen. I need to scroll down manually.

**Comment by tung2744** [comment]
@louischan-oursky Updated, thanks!

@fungc-io This is the updated colors.
Screenshot 2023-08-16 at 12 51 51 PM

**Pull request merged**